core/src/test/java/org/hibernate/ogm/backendtck/associations/collection/manytomany/ManyToManyTest.java

```diff
@@ -135,7 +135,7 @@ private void delete(Session session, Object... entities) {
 	}
 
 	private int expectedAssociationNumber() {
-		if ( TestHelper.getCurrentDialectType().equals( GridDialectType.NEO4J ) ) {
+		if ( TestHelper.getCurrentDialectType().equals( GridDialectType.NEO4J ) || TestHelper.getCurrentDialectType().equals( GridDialectType.NEO4J_REMOTE )) {
```

Comment:
See here:
https://github.com/hibernate/hibernate-ogm/pull/687/commits/090fbd951a86bc485cb0683e9ba794ac8917b70b#diff-0d9a048ba0f413c22966f840a46688a1L75
and here:
https://github.com/hibernate/hibernate-ogm/pull/687/commits/090fbd951a86bc485cb0683e9ba794ac8917b70b#diff-ca96314f78e37673af07f179b8c1f69eL138
or here:
https://github.com/hibernate/hibernate-ogm/pull/687/commits/090fbd951a86bc485cb0683e9ba794ac8917b70b#diff-c680f98d971b14f510ea306001b55d4aL120

Just a matter of consistency.